src/rgw/rgw_rest_s3.cc

```diff
@@ -684,6 +686,9 @@ void RGWPutBucketTags_ObjStore_S3::send_response()
 
 void RGWDeleteBucketTags_ObjStore_S3::send_response()
 {
+  if (op_ret == 0){
```

Comment:
Style: should be `if (...) {`. Please fix here and in other places.

Reply:
Sure.. Fixed  it